### PR TITLE
Add fpla admins as adoption code owners

### PR DESCRIPTION
**Pull request by tomaszpowroznik**
Add fpla admins as adoption code owners

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -150,6 +150,12 @@ k8s/**/common/family-public-law/namespace.yaml @hmcts/fpl-admins
 k8s/**/common/family-public-law/identity.yaml @hmcts/fpl-admins
 k8s/**/common/family-public-law/fpl-case-service.yaml @hmcts/fpl-admins
 
+### Adoption
+k8s/**/common/adoption/namespace.yaml @hmcts/fpl-admins
+k8s/**/common/adoption/identity.yaml @hmcts/fpl-admins
+k8s/**/common/adoption/adoption-frontend.yaml @hmcts/fpl-admins
```
Comment (adusumillipraveen) [inline CODEOWNERS]: This isn't allowed as this allows you to go live with these apps on prod without approvals.

Comment (tomaszpowroznik) [reply CODEOWNERS]: narrowed to non prod envs

File: CODEOWNERS [hunk 2]
```diff
@@ -150,6 +150,24 @@ k8s/**/common/family-public-law/namespace.yaml @hmcts/fpl-admins
 k8s/**/common/family-public-law/identity.yaml @hmcts/fpl-admins
 k8s/**/common/family-public-law/fpl-case-service.yaml @hmcts/fpl-admins
 
+### Adoption
+k8s/preview/common/adoption/namespace.yaml @hmcts/fpl-admins
+k8s/preview/common/adoption/identity.yaml @hmcts/fpl-admins
+k8s/preview/common/adoption/adoption-frontend.yaml @hmcts/fpl-admins
+k8s/preview/common/adoption/adoption-backend.yaml @hmcts/fpl-admins
+k8s/aat/common/adoption/namespace.yaml @hmcts/fpl-admins
```
Comment (timja) [inline CODEOWNERS]: only aat and prod are restricted, just list aat here as no need for the others unless you want to get the code review requests automatically?

Comment (tomaszpowroznik) [reply CODEOWNERS]: done, narrowed to aat only

**Pull request merged**